test/test_utils/__init__.py

```diff
@@ -415,6 +415,13 @@ def get_dlc_images():
         test_images = json.load(test_env)
     for dlc_test_type, images in test_images.items():
         if dlc_test_type == "sanity":
+            # TODO: Remove if re-enabling p2.8x tests
+            if os.getenv("EC2_GPU_INSTANCE_TYPE") == "p2.8xlarge":
+                non_gpu_images = []
+                for image in images:
+                    if "gpu" not in image:
+                        non_gpu_images.append(image)
+                return " ".join(non_gpu_images)
```

Comment:
Why is this necessary? p2.8xlarge is not assigned as an `EC2_GPU_INSTANCE_TYPE` for sanity tests.
There is no default instance type for sanity tests, right?

Reply:
This is unrelated to sanity tests - we are getting all the images from the sanity section of the test env file to unify w/ changeset logic. This can be refactored to be more clear as we update our changeset logic (i.e. add a section to the env file called all_images)

what this does is remove gpu images from dlc_images if EC2_GPU_INSTANCE_TYPE is p2.8x